lib/fast_jsonapi/object_serializer.rb

```diff
@@ -340,7 +340,9 @@ def validate_includes!(includes)
         return if includes.blank?
 
         parse_includes_list(includes).each_key do |include_item|
-          relationship_to_include = relationships_to_serialize[include_item]
+          relationships = relationships_to_serialize || {}
+          relationship_to_include = relationships[include_item]
```

Comment:
This is similar to what was done on the `fastjsonapi` gem that is now archived: https://github.com/Netflix/fast_jsonapi/pull/283.